util/eventbus/bus_test.go

```diff
@@ -64,6 +65,28 @@ func TestBus(t *testing.T) {
 	}
 }
 
+func TestSubscriberFunc(t *testing.T) {
+	synctest.Test(t, func(t *testing.T) {
+		b := eventbus.New()
+		defer b.Close()
+
+		c := b.Client("TestClient")
+
+		exp := expectEvents(t, EventA{12345})
+		eventbus.SubscribeFunc[EventA](c, func(e EventA) { exp.Got(e) })
+
+		p := eventbus.Publish[EventA](c)
+		p.Publish(EventA{12345})
```

Comment:
this test helper does t.Fatalf, but doesn't that run on the eventbus's `runWorker` goroutine, not this test goroutine?

Reply:
Good catch, thanks.